tools/shared.py

```diff
@@ -1219,6 +1219,7 @@ def reset(self):
     # Given some emcc-type args (-O3, -s X=Y, etc.), fill Settings with the right settings
     @classmethod
     def load(self, args=[]):
+      global WASM_TARGET, LLVM_TARGET
```

Comment:
this is worrysome. how about an env var for this, `EMCC_EXPERIMENTAL_USE_LLD`? then it would do this during startup in `shared.py`, which seems cleaner.